Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -171,7 +172,12 @@ - (instancetype)initWithConfiguration:(BPKCalendarConfiguration *)configuration
 - (void)setup {
     self.gregorian = [[NSCalendar alloc] initWithCalendarIdentifier:NSCalendarIdentifierGregorian];
 
+    self.calendarWrapperView = [UIView new];
     self.calendarView = [[FSCalendar alloc] initWithFrame:CGRectZero];
+    self.calendarView.clipsToBounds = NO;
+    self.calendarView.subviews[0].clipsToBounds = NO;
```

Comment:
We should annotate this with a comment like

```objective-c
// HACK: This relies on messing with the internal view hierarchy of `FSCalendar` and is liable to break.
```

Furhter we need to ensure snapshot tests fail if FSCalendar changes in a way that will break our assumptions about the view hiearchy.

Lastly, do we know that the subview orders are stable within FSCalendar? Is there a more stable way do write this logic i.e. by identifying the relevant views based on their class, tag, or something? 

Las

Reply:
I think the best way would be to iterate through subviews until you get to a specific class. There should be a max depth we go to in case we don't find the target view.
WDYT?

Reply:
I was thinking something less extreme, maybe just do one level but it means we can handle if the index changes?